mbenfowler/sound-stash
# Feature/add home page

## What I did:
<img width="485" alt="Screenshot 2023-07-23 at 5 25 29 PM" src="https://github.com/mbenfowler/sound-stash/assets/32660020/4f095169-e405-40ac-839f-3b4a4f7dfe9d">

- Added initial route for our `home` path (`'/'`) and populated it with our recommended list of albums!
- If user has no collection yet, it will fetch 5 albums from what we have determined is our search query for that
- If the user has a collection, the initial useEffect will instead grab a single album ID from their collection, fetch that album data for its genre, then fetch again 5 albums that share the genre
- Clicking on one of the albums in this recommendation list will navigate it you to its own url (no content here yet)
- Design only for mobile so far
- I used that first purplish palette, but it's likely we'll need to update some colors as it all comes together
- Not sure what font we're using yet so just kept it standard for now

## Testing Notes/Extras: 
- run the app from terminal: `npm start`
- observe the mock user data in `userData.js` - is this close to how our actual user data will be formed?
- the user hardcoded initial is the one with no collection yet
- Since they do not have a collection, on page load you should see "Albums trending today" and 5 trending albums
- clicking anywhere on an album will navigate you to its corresponding URL

- change the user in the code to the second user; this one has a single album id in its `collection`
- reload the page and you should now see "Recommended albums for you", along with 5 albums of the same genre
  - please confirm that if our fetch request contains an empty `genre` param we still get albums
  - should we handle this in a different way?
- clicking on an album takes you to its corresponding URL

Did this break anything?


Type of change


Checklist:


What's next?
Putting it all together!
More styling
